CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 ### Added
 
 - Automate python dependencies upgrade with pyup.
+- Create command bin/pytest.
```

Comment:
```suggestion
- Add `bin/pytest` script
```
Make the same change for the commit title and PR title.

Reply:
please

Reply:
Done my lord.